mlflow/metrics/utils/metric_definitions.py

```diff
@@ -249,3 +249,64 @@ def _rougeLsum_eval_fn(eval_df, metrics):
             scores=scores,
             aggregate_results=standard_aggregations(scores),
         )
+
+#TODO: add basic metrics definitions
+def _mae_eval_fn(eval_df, metrics):
+    if "target" in eval_df:
+        from sklearn.metrics import mean_absolute_error
+
+        y_pred = eval_df["prediction"]
+        y = eval_df["target"]
+
+        mae = mean_absolute_error(y, y_pred)
+        return MetricValue(aggregate_results={"": mae})
```

Comment:
Also we decided to use the same name for aggregate name as metric instead of empty string: look [here](https://github.com/mlflow/mlflow/pull/9594/files#r1323668187)